section_test.go

```diff
@@ -30,3 +32,122 @@ func TestSectionKeys(t *testing.T) {
 		t.Error("expected 'key3' to be in the list of keys")
 	}
 }
+
+func TestMergeSection(t *testing.T) {
+	config1Str := `
+	global = "global value";
```

Comment:
There are some indenting inconsistencies in these two config values, would you be able to fix those up?